agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4440,32 +4440,41 @@ beerocks::message::sWifiChannel slave_thread::channel_selection_select_channel()
             continue;
         }
         for (uint8_t i = 0; i < beerocks::message::SUPPORTED_CHANNELS_LENGTH; i++) {
-            auto channel         = hostap_params.supported_channels[i];
-            auto operating_class = wireless_utils::get_operating_class_by_channel(
-                channel.channel, static_cast<beerocks::eWiFiBandwidth>(channel.channel_bandwidth));
+            auto channel        = hostap_params.supported_channels[i];
+            auto bw             = static_cast<beerocks::eWiFiBandwidth>(channel.channel_bandwidth);
+            auto center_channel = wireless_utils::get_5g_center_channel(channel.channel, bw, true);
```

Comment:
Doesn't `hostap_params.supported_channels` also contain 2.4G channels?
On line 4445 you're calling `wireless_utils::get_5g_center_channel()` which will work fine with 2.4G channels as well, but if it's used not only for 5G channels, maybe the name should be changed to `get_center_channel`.

Reply:
The reason I called it `get_5g_center_channel` is because there is no use for center channel for 2.4G band. I can change the name to `get_center_channel` but then it might be confusing and cause people to use it when they shouldn't..

Reply:
It'll work just fine for 2.4G channels (simply doing nothing), but since we're using it for all channels (not just 2.4G), I think that `get_center_channel` is a more appropriate name.